packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -81,9 +81,17 @@ const releaseActionController = {
     });
     const query = await permissionsManager.sanitizeQuery(ctx.query);
```

Comment:
is the groupBy actually sanitized? as it's not a query parameter that we normally use in strapi.

Would it work the same if the wsanitization is done after replacing the `groupBy` with  the `sort`?